### PR TITLE
1851 tags clickable

### PR DESCRIPTION
**Pull request by mcdead154**
## 🍰 Pullrequest

Added feature, clickable tags below post, same as hashtags are.
### Issues

- relates #1851 

### Todo

- [X] None
image

### CODE REVIEW

File: webapp/components/Hashtag/Hashtag.vue [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <template>
-  <ds-tag>#{{ id }}</ds-tag>
+  <ds-tag>
+    <a v-bind:href="`/?hashtag=${id}`" target="_blank">#{{ id }}</a>
```
Comment (roschaefer) [inline webapp/components/Hashtag/Hashtag.vue]:
```suggestion
    <nuxt-link :to="hashtagUrl">#{{ id }}</nuxt-link>
```
and implement a `hashtagUrl` computed property :wink:

**Comment by mcdead154** [comment]
@roschaefer I have tried nuxt link, but hashtag opens a new tab on click, and I haven't find a way for nuxt link to open a blank tab, so I have used an <a> tag instead

**Comment by roschaefer** [comment]
`nuxt-link` does not need to open in a new tab. It's used when we link within our app. If we link to somewhere outside of our app, we would use `<a href="...">`. Among other things it does link prefetching and we would loose that optimization.

**Comment by roschaefer** [comment]
It might be that the editor uses `<a>` tags. But that's not on purpose. It's because we did not figure out yet how to render `nuxt-link`s inside an editor node.

**Pull request merged**